docs/examples/ref_destroy.rb

```diff
@@ -71,12 +71,24 @@
   action :destroy
 end
 
-aws_route_table 'ref-main-route-table' do
-  action :destroy
+# TODO fix this so it creates a new default route table and detaches an existing one
+aws_vpc 'ref-vpc' do
+  main_routes '0.0.0.0/0' => :internet_gateway
 end
 
+# The only call that changes the MAIN route table is the
+# 'create route table' call - there is no update except
+# through the console, and you cannot delete the main
+# route table.  Thus, we must purge the VPC which deletes
+# all non-main route tables, and then the delete
+# vpc call automatically deletes the main route table
+
+# aws_route_table 'ref-main-route-table' do
+#   action :destroy
+# end
+
 aws_vpc 'ref-vpc' do
-  action :destroy
+  action :purge
```

Comment:
Do we know why `purge` manages to destroy the main route table while `destroy` does not?  It seems like `destroy` ought to have been sufficient here.